cmd/search/main.go

```diff
@@ -84,6 +87,12 @@ func main() {
 	flag.StringVar(&opt.BugzillaTokenPath, "bugzilla-token-file", opt.BugzillaTokenPath, "A file to read a bugzilla token from.")
 	flag.StringVar(&opt.BugzillaSearch, "bugzilla-search", opt.BugzillaSearch, "A quicksearch query to search for bugs to index.")
 
+	// jira
+	flag.StringVar(&opt.JiraURL, "jira-url", opt.JiraURL, "The URL of a bugzilla server to index bugs from.")
+	flag.StringVar(&opt.JiraTokenPath, "jira-token-file", opt.JiraTokenPath, "A file to read a bugzilla token from.")
+	flag.StringVar(&opt.JiraSearch, "jira-search", opt.JiraSearch, "A quicksearch query to search for bugs to index.")
+	//
```

Comment:
Copy-n-paste issues (Jira != Bugzilla) and an empty comment.

Reply:
Fixed